envex/env_wrapper.py

```diff
@@ -76,6 +76,7 @@ def __init__(
             base_path=base_path,
             engine=engine,
             mount_point=mount_point,
+            timeout=kwargs.get("timeout", None),
```

Comment:
The class to which this is passed handles None as a special case, and setting it here means it would always pass a value that the called class will use regardless of its special handling